collect/src/main/java/com/vgscollect/androiddemo/samples/cname/CnameActivity.kt

```diff
@@ -0,0 +1,65 @@
+package com.vgscollect.androiddemo.samples.cname
+
+import android.os.Bundle
+import android.util.Log
+import android.view.Gravity
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.setPadding
+import com.verygoodsecurity.vgscollect.core.HTTPMethod
+import com.verygoodsecurity.vgscollect.core.VGSCollect
+import com.verygoodsecurity.vgscollect.core.VgsCollectResponseListener
+import com.verygoodsecurity.vgscollect.core.model.network.VGSRequest
+import com.verygoodsecurity.vgscollect.core.model.network.VGSResponse
+import com.verygoodsecurity.vgscollect.widget.VGSCardNumberEditText
+import com.vgscollect.androiddemo.R
+import kotlinx.android.synthetic.main.activity_layout.*
+
+class CnameActivity : AppCompatActivity() {
+
+    private lateinit var vgsCollect: VGSCollect
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_layout)
+
+        // Init collect, configure custom host name in <a href="https://dashboard.verygoodsecurity.com/">VGS dashboard</a> before using it.
```

Comment:
I think would be better if we place comments under methods. It will be more comfortable for users to read a description that could be read without scrolling.
Apply it on all comments please.